includes/utilities.inc

```diff
@@ -284,7 +282,7 @@ function islandora_multi_importer_retrieve_remote_file($url, $destination = NULL
     $parsed_url = parse_url($url);
     $mime = 'application/octet-stream';
     if (!isset($destination)) {
-      $path = file_build_uri(drupal_basename($parsed_url['path']));
+      $path = _imi_build_file_storage_uri(drupal_basename($parsed_url['path']), 'remote-files');
```

Comment:
So is 'remote-files' fixed here? if we are going to fix paths for people then we should add an imi- as prefix to avoid conflicts with other modules (not likely but better to be prepared)

Reply:
'remote-files' is a subdirectory of the IMI temp files directory, so it seemed unnecessary to namespace it. 'csv' is also created within this directory, and not namespaced for the same reason.
Of course the IMI temp directory could be at the root of private://, which would eliminate that implied namespacing. 
Do you think it best to prefix all IMI temp subdirectories with `imi_`?

Reply:
I feel we could prefix all subdirectories with an imi-...